src/axios-auth.ts

```diff
@@ -1,5 +1,10 @@
 import axios from 'axios'
 import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
+// import { createPinia, setActivePinia } from 'pinia'
+// import { useAuthenticationStore } from '@/stores'
+
+// setActivePinia(createPinia())
+// const authenticationStore = useAuthenticationStore()
```

Comment:
This was my original code but it didn't pick up the account id, possibly because the session storage wasn't set yet.

The new code below works. I'll try to clean it up a bit.

Ignore the console log on line 24 -- I'm still testing!